src/fontra/views/editor/visualization-layer-definitions.js

```diff
@@ -172,6 +177,29 @@ registerVisualizationLayerDefinition({
   },
 });
 
+registerVisualizationLayerDefinition({
```

Comment:
I think we need two visualisation layers:
- one for edit mode, *not* userSwitchable, always on
- one for non-edit mode, userSwitchable, off by default ("Glyph lock icon for non-editing glyphs")

Reply:
done.